end-to-end-tests/src/main/java/com/amazonaws/blox/integ/BloxTestStack.java

```diff
@@ -29,20 +31,68 @@
  * of this whole stack, so this will probably be removed soon.
  */
 public class BloxTestStack {
-  private CloudFormationClient cloudFormationClient = CloudFormationClient.create();
-  private DynamoDBClient dynamoDBClient = DynamoDBClient.create();
-  private ECSClient ecsClient = ECSClient.create();
-  private LambdaAsyncClient lambdaClient = LambdaAsyncClient.create();
+  private String region;
+  private String credentialProfile;
+  private String bloxEndpoint;
+  private ProfileCredentialsProvider profileCredentialsProvider;
 
-  private CloudFormationStacks stacks = new CloudFormationStacks(cloudFormationClient);
+  private CloudFormationClient cloudFormationClient;
+  private DynamoDBClient dynamoDBClient;
+  private ECSClient ecsClient;
+  private LambdaAsyncClient lambdaClient;
 
-  private final DataServiceWrapper dataService =
-      new DataServiceWrapper(lambdaClient, stacks, dynamoDBClient);
-  private final ECSClusterWrapper ecs = new ECSClusterWrapper(ecsClient, stacks);
+  private CloudFormationStacks stacks;
 
-  public DataService createDataService() {
-    // TODO: Replace with frontend once we have it wired up
-    return dataService.createDataService();
+  private ECSClusterWrapper ecs;
+
+  private Blox blox;
+
+  public BloxTestStack(String region, String credentialProfile, String bloxEndpoint) {
+    this.region = region;
+    this.credentialProfile = credentialProfile;
+    this.bloxEndpoint = bloxEndpoint;
+
+    this.profileCredentialsProvider =
+        ProfileCredentialsProvider.builder().profileName(this.credentialProfile).build();
+
+    this.cloudFormationClient =
+        CloudFormationClient.builder()
+            .credentialsProvider(profileCredentialsProvider)
+            .region(Region.of(this.region))
+            .build();
+
+    this.dynamoDBClient =
+        DynamoDBClient.builder()
```

Comment:
I don't think we need this client anymore, since this PR also deletes the code that directly interacted with DDB to clear tables.

Reply:
I was originally deleting this code because we are planning for delete environment.  However, it looks DeleteEnvironment is not ready yet. I will actually revert the dynamodb clean table code back.